pkg/client/ocm/client.go

```diff
@@ -487,7 +487,25 @@ func (c *client) CreateMachinePool(clusterID string, machinePool *clustersmgmtv1
 	return createdMachinePool, nil
 }
 
-func (c client) GetQuotaCosts(organizationID string, fetchRelatedResources, fetchCloudAccounts bool) (*amsv1.QuotaCostList, error) {
+// QuotaCostRelatedResourceFilter represents the properties of the related resource, associated
+// to each quota cost, that can be used to filter the result of the get quota costs request
+type QuotaCostRelatedResourceFilter struct {
+	ResourceName *string
```

Comment:
Can we document the behavior about one of the attributes of the filter being nil meaning it is not evaluated?